### ISSUE TITLE
how to update the package?

### ISSUE THREAD

**Issue by mygzi** [syncup-chat/SDK#1]
I've committed the regex fix (60895c6b0b0ce8327afd25a8f649028be95254c0) and created a meteor developer account (as mygzi). 

When I try to 'meteor publish', I get:

> While pre-publishing package hblockhus:syncup:
> error: You are not an authorized maintainer for this package.

Is there a way to be added as a maintainer?

**Comment by hblockhus** [comment]
I'm not sure, I tried making an organization and adding mygzi to it in my
meteor developer account page. Maybe that will let us publish from two
accounts

On Saturday, July 2, 2016, mygzi notifications@github.com wrote:

> I've committed the regex fix (60895c6
> https://github.com/syncup-chat/SDK/commit/60895c6b0b0ce8327afd25a8f649028be95254c0)
> and created a meteor developer account (as mygzi).
> 
> When I try to 'meteor publish', I get:
> 
> While pre-publishing package hblockhus:syncup:
> error: You are not an authorized maintainer for this package.
> 
> Is there a way to be added as a maintainer?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> https://github.com/syncup-chat/SDK/issues/1, or mute the thread
> https://github.com/notifications/unsubscribe/AES1LbfX468fbpwUVFB3Gml9LHFmgzziks5qRwELgaJpZM4JDxqe
> .

## 

Heather Blockhus
iOS Developer | Syncup Communications Inc.

**Comment by mygzi** [comment]
it seems that to do this, the package has to be renamed to reflect the organization, so I created a new one that in theory we can both update:

```
$ meteor publish --create
Published syncupinc:syncup@1.0.0. 
```

apps will need to be updated to use this new package instead.

**Comment by hblockhus** [comment]
Ok, not at home now. Just need to change the .meteor/packages file for the
new package. Or run meteor add wherever you're hosting it from

On Saturday, July 2, 2016, mygzi notifications@github.com wrote:

> it seems that to do this, the package has to be renamed to reflect the
> organization, so I created a new one that in theory we can both update:
> 
> $ meteor publish --create
> Published syncupinc:syncup@1.0.0. <javascript:_e(%7B%7D,'cvml','syncupinc:syncup@1.0.0.');>
> 
> apps will need to be updated to use this new package instead.
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> https://github.com/syncup-chat/SDK/issues/1#issuecomment-230128954, or mute
> the thread
> https://github.com/notifications/unsubscribe/AES1LVyOxeZjumYV5T8ESkveijSHFV96ks5qRwUHgaJpZM4JDxqe
> .

## 

Heather Blockhus
iOS Developer | Syncup Communications Inc.
